gsy_framework/constants_limits.py

```diff
@@ -166,8 +166,8 @@ class IAASettings:
         MARKET_TYPE = SpotMarketTypeEnum.ONE_SIDED.value
         MARKET_TYPE_LIMIT = RangeLimit(1, 2)
 
-        BID_OFFER_MATCH_TYPE = BidOfferMatchAlgoEnum.PAY_AS_BID.value
-        BID_OFFER_MATCH_TYPE_LIMIT = RangeLimit(1, 3)
+        ORDERS_MATCH_TYPE = OrdersMatchAlgoEnum.PAY_AS_BID.value
```

Comment:
How about providing backwards compatibility and add the old name, like this:
```suggestion
        ORDERS_MATCH_TYPE = OrdersMatchAlgoEnum.PAY_AS_BID.value
        # for backwards compatibility:
        BID_OFFER_MATCH_TYPE = ORDERS_MATCH_TYPE
```
(same for `EXPORT_ORDERS_TRADE_HR `)